doc/general/cash-register-integration/cash-register-integration-failure-scenarios.md

```diff
@@ -6,7 +6,31 @@ title: Failure scenarios
 ### Failure Scenarios
 This Chapter describes the different scenarios of failure when using the fiskaltrust.Middleware and how to handle them.
 
-#### Failure of the fiskaltrust.Middleware
+On a high level, there are three different failure scenarios that should be considered that are described in more detail in the following sections:
+1. **Signature Cration Unit not reachable or failing:** When the SCU is either not reachable or returns an error (e.g. because of an outage of the wrapped signing device or service), the Middleware switches to the _failed mode_, i.e. a circuit breaker mode that doesn't execute further calls towards the SCU, which _must_ be reset by sending a _zero receipt_.
+2. **Middleware not reachable or failing**: When the cash register doesn't receive any response from the Middleware (e.g. because of a network or server outage), receipts _must_ be re-sent as soon as the Middleware is reachable again, using the flag `0x0000000000010000` to switch to the _late-signing mode_. If required in the respective country, the Middleware will automatically take care of late-signig the receipts then. When the receipts have been successfully re-sent, failure-mode _must_ be left by sending a _zero receipt_.
+3. **Cash register outage**: When the whole cash register/POS system is not working anymore (e.g. because of a power outage or a system failure) and the local fiscalization laws require the tracking of handwritten receipts (like e.g. in Germany) in that case, receipts _may_ be re-sent to the Middleware after the systems are back online with the _handwritten ftReceiptCaseFlag_ `0x0000000000080000`. This is optional, but recommended for consistentency in case of an audit.
+
+#### Signature Cration Unit not reachable or failing
+If the communication to the SCU fails (e.g. when the secure Signature Creation Device is not reachable), the POS System can continue to operate until the SCU is accessible again. Receipts created in a state where no communication is possible with the SCU are protected by the security mechanism of fiskaltrust. The fiskaltrust.Middleware will respond with the ftState = `0x02` "SCU communication failed". The POS-System receives the response and processes the data it contains. For following Requests no more communication attempts are done to avoid long waiting times for each Receipt request/Receipt response sequence.
```

Comment:
Maybe we should also focus on the fact that the Middleware does all things necessary to still be compliant instead of writing that the communication is not performed anymore. I guess if we have a market that we would need to check communication we would probably still do so. 

TL;DR let's focus on the middleware capabilities that this mode keeps the receipt chain compliant by still allowing to handle outages of the local signing device

Reply:
I had the feeling that this was already "mentioned" in a way, but you're right, makes more sense to be explicit about this - done